### ISSUE TITLE
should we migrate modularity translation?

### ISSUE THREAD

**Issue by Jibec** [fedora-modularity/ModulemdTranslationHelpers#5]
Hello,

we have this project in Zanata: https://fedora.zanata.org/project/view/fedora-modularity-translations
and we are moving to Weblate: https://fedoraproject.org/wiki/L10N_Move_to_Weblate

would you like to migrate it?
what the status of translation support for modularity?

**Comment by OrionStar25** [comment]
Hello @Jibec,

> we have this project in Zanata: https://fedora.zanata.org/project/view/fedora-modularity-translations
> and we are moving to Weblate: https://fedoraproject.org/wiki/L10N_Move_to_Weblate
> 
> would you like to migrate it?

The code I've written isn't specifically for Zanata. It only depends on `Babel catalogs` or `portable object files (.po)` for now. Looks like Webplate is going to stay for another decade at least. I'll try to integrate this library with Webplate then.

> what the status of translation support for modularity?

This code isn't deployed yet. @sgallagher can give you more information on this.

**Comment by Jibec** [comment]
Well, could you please, commit existing translation in your repo and generate the new pot?
I'll handle the Weblate configuration for you.
For deployment, I would be interested indeed to know when it will reach end users, but if pot generation works, we can plug Weblate on it.

**Comment by OrionStar25** [comment]
`.pot` generation does work using [this](https://github.com/fedora-modularity/ModulemdTranslationHelpers#modulemdtranslationhelpersget_translation_catalog_from_index). But only @sgallagher can commit to this repo since I don't have push access to it.

**Comment by sgallagher** [comment]
@Jibec Yes, I do want the translations ported, but I was overcommitted for the last few months and didn't get around to deploying the translation work yet. I'll try to regenerate the contents manually for you this week.

**Comment by Jibec** [comment]
Is this project still active?
I locked Fedora Modules in Zanata, because it will be removed in a few weeks: https://fedoraproject.org/wiki/Changes/Zanata_removal

I would love to help you, but need to understand how. Please let me know!
